packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Snippets/ClientUriBuilderSnippets.cs

```diff
@@ -23,8 +23,11 @@ public static InvokeMethodExpression AppendQuery(this ScopedApi<ClientUriBuilder
         public static InvokeMethodExpression AppendQuery(this ScopedApi<ClientUriBuilderDefinition> uriBuilder, ValueExpression name, ValueExpression value, string? format, bool shouldEscape)
             => uriBuilder.Invoke("AppendQuery", [name, value, Literal(format), Literal(shouldEscape)]);
 
+        public static InvokeMethodExpression AppendQueryDelimited(this ScopedApi<ClientUriBuilderDefinition> uriBuilder, ValueExpression name, ValueExpression value, string delimiter, string? format, bool shouldEscape)
+            => uriBuilder.Invoke("AppendQueryDelimited", [name, value, Literal(delimiter), Literal(format), Literal(shouldEscape)]);
+
         public static InvokeMethodExpression AppendQueryDelimited(this ScopedApi<ClientUriBuilderDefinition> uriBuilder, ValueExpression name, ValueExpression value, string? format, bool shouldEscape)
```

Comment:
Should we just make delimiter an optional parameter rather than add new overload?

Reply:
yeah that's a good suggestion. Fixed.